README.md

```diff
@@ -23,15 +23,17 @@ plugins {
 }
 
 dependencies {
-  annotationProcessor "com.uber.nullaway:nullaway:0.9.1"
+  errorprone("com.uber.nullaway:nullaway:0.9.1")
```

Comment:
Should we also update the suggested version of `net.ltgt.errorprone` while we are at it?  I noticed the latest version is 2.0.1, so maybe 0.6 is quite old now

Reply:
Also, lower down where it says "the `annotationProcessor` line loads NullAway", that should also be changed to "the first `errorprone` line" or something like that

Reply:
yeah, that version is ancient, honestly I didn't really notice, or worry about fixing that, but maybe that's why this was written this way. Maybe the better change would just be to point to the plugin's documentation as it actually covers all of this...